packages/notebook/style/base.css

```diff
@@ -47,6 +47,14 @@
   min-height: var(--jp-notebook-scroll-padding);
 }
 
+.jp-MainAreaWidget-ContainStrict .jp-Notebook * {
+  contain: strict;
+}
+
+.jp-Notebook-render * {
+  contain: none !important;
+}
```

Comment:
@echarles do you remember where `jp-Notebook-render` was used? I cannot find it in the codebase - this style appears unused.

Reply:
we made a few iterations with the `contain` rule probably 2 years ago. If `.jp-Notebook-render` is not used anymore, it is safe to simply remove it from the base.css file I would say.